recipes/conversation_view/message_input/message_input.vue

```diff
@@ -554,6 +555,12 @@ export default {
     },
   },
 
+  watch: {
+    modelValue (newValue) {
```

Comment:
am not sure if we should be checking current value and compare to newValue here.

Reply:
No don't think this is necessary. Vue itself is smart enough to know if the value was changed or not.